index.html

```diff
@@ -461,5 +259,7 @@ <h2 class="contact-text">Contact me</h2>
         </div>
       </footer>
     </section>
+
+    <script src="popup.js"></script>
```

Comment:
The pop-up image should be the full width of the container on desktop view. Moreover, the technologies list and buttons should be on the right side of the container, and the text should be on the left side. Please follow the design for the desktop view. Kindly see the screenshot below and fix the design issues.
![desktop](https://user-images.githubusercontent.com/35504149/132958217-0306c60a-245e-4f48-b164-d4786ef13c56.png)